docs/other_resources/in-process-providers/evaluators/fractional_evaluation.md

```diff
@@ -0,0 +1,193 @@
+# Fractional Evaluation
+
+This evaluator allows to split the returned variants of a feature flag into different buckets,
+where each bucket can be assigned a percentage, representing how many requests will resolve to the corresponding
+variant. The sum of all weights must be 100, and the distribution must be performed by using the value of a referenced
+from the evaluation context to hash that value and map it to a value between [0, 100]. It is important to note
+that evaluations MUST be sticky, meaning that flag resolution requests containing the same value for the
+referenced property in their context MUST always resolve to the same variant. For calculating the hash value of the
+referenced evaluation context property, the [MurmurHash3](https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp)
+hash function should be used. This is to ensure that flag resolution requests yield the same result,
+regardless of which implementation of the in-process flagd provider is being used.
+
+The implementation of this evaluator should accept the object containing the `fractionalEvaluation` evaluator
+configuration, and a `data` object containing the evaluation context. The evaluator configuration should be an
+array containing at least two items, with the first item being a `string`
+value representing the target property to base the distribution of values on, and the remaining items
+being `arrays` with two values, with the first being `string` item representing the name of the variant, and the
```

Comment:
```suggestion
array containing at least two items, with the first item being a [json logic variable declaration](https://jsonlogic.com/operations.html#var)
specifying the target property to base the distribution of values on, and the remaining items
being `arrays` with two values, with the first being `string` item representing the name of the variant, and the
```

Reply:
I didn't include the option to omit the first parameter for now. We can do that if we want, but I'd like to implement just this for now, and if handling that shorthand is easily implemented we can mention it here.